project/Dependencies.scala

```diff
@@ -32,7 +32,7 @@ object Dependencies {
 
   val sslConfigVersion = "0.4.2"
 
-  val scalaTestVersion = "3.1.4"
+  val scalaTestVersion = "3.2.3"
```

Comment:
looks like org.scalatestplus:mockito-3-3 was not released for scalaTest 3.2.3 (https://repo1.maven.org/maven2/org/scalatestplus/mockito-3-3_2.12/) - perhaps we should update to org.scalatestplus:mockito-3-4?

Reply:
there's been some discussion on when to update to scalatest 3.2 in #29368

Reply:
Thanks for the context @raboof !
Two options for moving forward:
 - I revert back the bump of ScalaTest for having this PR merged
 - we are in 2021 and we can possibly unblock the bump since @patriknw said:
  > we should be careful to not upset users with forcing another update this soon again

as you prefer.

Reply:
can we use a different version of ScalaTest for Scala 3?

Reply:
no, `3.2.3` is the only single release of scalatest published for Scala 3.
https://repo1.maven.org/maven2/org/scalatest/scalatest-core_3.0.0-M3/

Reply:
Sorry, my question was unclear. I mean, can we use ScalaTest 3.1 for Scala 2 and ScalaTest 3.2 for Scala 3?

Reply:
well, yes, this is an option.
If this unblocks this PR I can do that, but I think is much cleaner for Akka to keep using a single version of ScalaTest.
Also, I will probably have to switch also the Mockito and ScalaCheck versions, is it acceptable?

Reply:
I don't think we have the bandwidth to investigate what an incompatible bumb of ScalaTest would mean for our Scala 2.12 and 2.13 users and especially the full Akka ecosystem of dependent libraries at this moment.
Quickest way forward for this PR would be to use a 3.2.3 only for Scala 3.

Reply:
Ok, reverted the bump of ScalaTest and ScalaCheck (will do the switch after #29932 to avoid conflicts).
Kept the bump of Mockito though.